packages/webcomponents/src/components/business-forms/payment-provisioning/document-uploads/business-document-upload-form-step.tsx

```diff
@@ -59,7 +59,7 @@ export class BusinessDocumentFormStep {
     this.formLoading.emit(true);
     try {
       const response: IApiResponse<IBusiness> = await this.api.get(this.businessEndpoint);
-      this.business = { ...new Business(response.data) };
+      this.business = { ...new Business(response.data) } as Business;
```

Comment:
I'm sure there is a reason, but why can't this be `new Business(response.data)` without spreading and casting? Does it need to be spread onto a new object to trigger a re-render? If so it's kinda odd, I'd think that the constructor for `Business` would return a new object and trigger a re-render too 🤔

Reply:
Yes, I agree it’s a bit odd. The original code was written this way because simply using new Business was causing a bug at the time, and this was the solution that worked. I’ll test using new Business again to see if the issue persists. If it does, I’ll add a comment explaining why this approach is necessary

Reply:
Oh hey I have some context to add here: 

The reason we instantiated and spread like this is because when we would send the instantiated data without spreading it, we would fail the POST request because for some reason the constructor of the class was being sent along with the payload.... It was a weird bug. At the time we just decided we were fine with spreading the instantiated class and just moving on. 

Maybe we can fix this properly now that we are sending the results of the class getter method as the payload, instead of the entire class / object itself. 🤔

Reply:
Oh exactly, thanks Colgan. That was the case. Now with the new approach, I tested it and the spread is not necessary anymore! 🙌 💯